### ISSUE TITLE
Apply changes to lesson template specified in fair4rs-lesson-setup

### ISSUE THREAD

**Issue by tdjames1** [FAIR2-for-research-software/FAIR4RS-Packaging#26]
Please could you make sure you've applied the adjustments to the Carpentries lesson template that are indicated in https://github.com/RSE-Sheffield/fair4rs-lesson-setup (apply modifications from the patch file and update metadata)?

**Comment by f-allian** [comment]
Thanks for the flag @tdjames1. 

@christopher-wild can you take a look at this? I tried setting it up but had no luck, I'm unsure if it's a problem with my R-studio.

**Comment by tdjames1** [comment]
Hi @christopher-wild just a reminder to make sure the changes in the patch file (e.g. updates to CODE_OF_CONDUCT, CONTRIBUTING, addition of acknowledgements etc) have been applied to your repo? You can grab the files from one of the other repos if that's easier. 🙏

**Comment by tdjames1** [comment]
Yes, it was designed to be applied directly after cloning the template - the changes you've made to the repo interfere with the patch applying cleanly.
